### PR TITLE
Instructions for configuring a service account

### PR DESCRIPTION
**Pull request by alahwa**
Also fix the typo

### CODE REVIEW

File: bigquery/deploy/README.md [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
+    * Switch projects to the one containing your bigquery table and add an IAM
+    role for your new service account
+      * Grant the `BigQuery > BigQuery Job Viewer` role to the Service account
+  permissions.
   * Create cluster
     * Go to https://console.cloud.google.com/kubernetes/list and click `Create Cluster`
     * Change name to `elasticsearch-cluster`
     * Change `Machine type` to `4 vCPUs`. (Otherwise will get Insufficient CPU error.)
-    * Expand `More` -> Click on `Set access for each API` -> Change `BigQuery` to enabled.
+    * Expand `More`
```
Comment (melissachang) [inline bigquery/deploy/README.md]: * Expand `More` -> Select the service account you just created.

File: bigquery/deploy/README.md [hunk 2]
```diff
@@ -1,11 +1,25 @@
 ## Running on GKE
 
 * Set up the Kubernetes environment
+  * Create a service account with access to your table
```
[the remark this answers is not in the document]
Comment (alahwa) [reply bigquery/deploy/README.md]: Done

File: bigquery/deploy/README.md (hunk 1 above)
Comment (alahwa) [reply bigquery/deploy/README.md]: Done

File: bigquery/deploy/README.md [hunk 3]
```diff
@@ -1,11 +1,36 @@
...
+      account to run a BigQuery query, which takes place while indexing the BigQuery tables into Elasticsearch. Note that [this project will be billed](https://github.com/DataBiosphere/data-explorer-indexers/blob/master/bigquery/indexer.py#L131) for the BigQuery query, not the project containing the BigQuery tables.
+    * Add the service account to the Google Group with read-only access to the
+    dataset
   * Create cluster
     * Go to https://console.cloud.google.com/kubernetes/list and click `Create Cluster`
     * Change name to `elasticsearch-cluster`
     * Change `Machine type` to `4 vCPUs`. (Otherwise will get Insufficient CPU error.)
-    * Expand `More` -> Click on `Set access for each API` -> Change `BigQuery` to enabled.
+    * Expand `More` and select the service account you just created.
+      * If referencing a bigquery dataset outside of your current project,
```
Comment (melissachang) [inline bigquery/deploy/README.md]: Sorry I wasn't clear, delete lines 30-33

**Pull request merged**